### PR TITLE
Clarify instructions on installing Extensibility Tools

### PR DESCRIPTION
**Pull request by dharmatech**

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/roslyn-infrastructure @jasonmalinowski 

:+1:

File: docs/contributing/Building, Debugging, and Testing on Windows.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 # Required Software
 
 1. [Visual Studio 2015 with Update 1](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 1_.
-2. Visual Studio 2015 Extensibility Tools. If you already installed Visual Studio, choose "Modify" from the Programs and Features control panel, and check "Visual Studio Extensibility".
+2. Visual Studio 2015 Extensibility Tools.
+ 
+    If you already installed Visual Studio, the Extensibility Tools can be added as follows: 
+    - Open Control Panel -> Programs and Features
+    - Select "Microsoft Visual Studio Professional 2015" and press the 'Change' button
```
Comment (jasonmalinowski) [inline docs/contributing/Building, Debugging, and Testing on Windows.md]: Or Community, or Enterprise.

**Comment by jasonmalinowski** [comment]
:+1:

**Comment by davkean** [comment]
Thanks for the contribution.

**Pull request merged**